drivers/d3d12/rendering_context_driver_d3d12.cpp

```diff
@@ -99,7 +99,7 @@ RenderingContextDriverD3D12::~RenderingContextDriverD3D12() {
 }
 
 Error RenderingContextDriverD3D12::_init_device_factory() {
-	uint32_t agility_sdk_version = GLOBAL_GET("rendering/rendering_device/d3d12/agility_sdk_version");
+	uint32_t agility_sdk_version = GLOBAL_GET(PropertyInfo(Variant::INT, "rendering/rendering_device/d3d12/agility_sdk_version", "1,10000,1,or_greater");
```

Comment:
I don't think it makes much sense for this to be a range.  It should match the version number of the DirectX Agility SDK included with Godot, not be set to an arbitrary value.

Right now, the version downloaded by the `install_d3d12_sdk_windows.py` script is `613`.

Valid versions can be found here: https://devblogs.microsoft.com/directx/directx12agility/

Reply:
The default value is already set to 613, but you can use a newer version if you wish. This is supposed to work without having to update Godot itself.

We could probably tweak the hint to disallow using versions older than the version Godot originally downloaded in its script, still.

Reply:
cc @RandomShaper @DarioSamo

Reply:
One could even want to use an older version, I think, for some reason. That said, probably the sanest is to constrain to `613` as the minimum and leave using a higher one to the user. If anything, a GUI slider would seem a bit off. So, something like this? (not sure if `or_less` makes sense with no slider visible instead of just allowing any integer value):
```c++
uint32_t agility_sdk_version = GLOBAL_GET(PropertyInfo(Variant::INT, "rendering/rendering_device/d3d12/agility_sdk_version", "613,10000,1,or_less,hide_slider");
```

Reply:
Done. The property hint range now only ensures the value is positive (the slider is hidden either way as it's an integer property, not a float).